docs/migrating-from-scalibr.md

```diff
@@ -0,0 +1,113 @@
+
+### Scanning a directory
+
+**osv-scalibr:**
+
+```sh
+scalibr --root /path/to/your/project
+```
+
+**osv-scanner:**
+
+```sh
+osv-scanner /path/to/your/project
+```
+
+### Selecting plugins
+
+OSV-Scanner has access to the full list of OSV-Scalibr plugins, though only a well tested subset of them are enabled by default in OSV-Scanner.
+
+In `osv-scalibr`, you can select which plugins to run using the `--extractors`, `--detectors` flags,
+or alternatively using the `--plugins` flag.
+
+**osv-scalibr:**
+
+```sh
+scalibr --plugins python/pip,go/gomod --detectors go/govulncheck /path/to/your/project
+```
+
+In `osv-scanner`, you can achieve the same by using the `--experimental-plugins` flag. This is an experimental feature.
+
+**osv-scanner:**
+
+```sh
+osv-scanner --experimental-plugins python/pip,go/gomod,go/govulncheck /path/to/your/project
+```
+
+`osv-scanner` also allows you to disable default plugins with `--experimental-disable-plugins`.
+
+For more details on manual plugin selection in `osv-scanner`, see the [manual plugin selection documentation](manual-plugin-selection.md).
+
+### Generating SPDX output
+
+`osv-scalibr` uses the `-o` flag to specify the output format and file. For example, to generate an SPDX JSON report:
+
+**osv-scalibr:**
+
+```sh
+scalibr -o spdx23-json=result.spdx.json /path/to/your/project
+```
+
+`osv-scanner` uses the `--format` flag to specify the output format and the output is written to standard output,
+and a separate `--output` flag if you wish to save the output into a file.
+
+**osv-scanner:**
+
+```sh
+osv-scanner --format spdx-2.3-json /path/to/your/project > result.spdx.json
+```
+
+For more details on `osv-scanner` output formats, see the [output documentation](output.md).
+
+## Flag Translation Table
+
+| `osv-scalibr` Flag                | `osv-scanner` Flag        | Notes                                                                                                 |
+| --------------------------------- | ------------------------- | ----------------------------------------------------------------------------------------------------- |
+| `--version`                       | `--version`               | `osv-scanner version`                                                                                 |
+| `--root`                          | `[directory]` (argument)  | `osv-scanner scan source [directory]`                                                                 |
+| `--result`                        | `--output`                | `osv-scanner --output <file>`                                                                         |
+| `-o`                              | `--format` and `--output` | e.g. `osv-scalibr -o spdx23-json=r.json` becomes `osv-scanner --format spdx-2.3-json --output r.json` |
+| `--plugins`                       | `--experimental-plugins`  |                                                                                                       |
```

Comment:
do we want to list some example plugins/extractors/detectors here?

Reply:
Made https://github.com/google/osv-scalibr/pull/1137 which I'll link to on this doc for the list of plugins available.